### PR TITLE
Added jackline bountysource id

### PR DESCRIPTION
**Pull request by schoeke**

### CODE REVIEW

File: _data/clients/jackline.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 name: Jackline
 url: https://github.com/hannesm/jackline
 tracking_issue: https://github.com/hannesm/jackline/issues/153
+bountysoure: 41164178
```
Comment (wiktor-k) [inline _data/clients/jackline.yml]: Shouldn't that be *bountysource* (missing *c*)?

Comment (bascht) [inline _data/clients/jackline.yml]: @wiktor-k Correct. Good find. :+1:

**Comment by bascht** [comment]
@schoeke Can you fix the one typo? I'm off for the holidays, but @ReneVolution will merge your PR later. :smile:

File: _data/clients/jackline.yml (hunk 1 above)
Comment (schoeke) [inline _data/clients/jackline.yml]: Ah, sorry. I was too tired apparently. -.-

**Comment by ReneVolution** [comment]
Thanks @schoeke 😄

**Pull request merged**